packages/ui/components/Form/Inputs/Dropdown.tsx

```diff
@@ -0,0 +1,125 @@
+import * as RadixSelect from '@radix-ui/react-select'
+import { CheckIcon, ChevronDownIcon, ChevronUpIcon } from '../../icons'
+import classnames from 'classnames'
+
+type DropdownRegularOption = {
+  type?: 'regular'
+  value: string
+  label: string
+}
+
+type DropdownRegularOptionProps = {
+  option: DropdownRegularOption
+}
+
+const DropdownRegularOption = ({ option: { value, label } }: DropdownRegularOptionProps) => {
+  return (
+    <RadixSelect.Item
```

Comment:
```suggestion
    <RadixSelect.Item
      aria-label={`Option ${label}`}
```

Reply:
I'm pretty sure Radix is already adding `aria-label`s 🤔 We should use https://storybook.js.org/addons/addon-screen-reader on Storybook to test it.

Reply:
that addon is not being maintained anymore. I've added "@storybook/addon-a11y" to test for accessibility. I have resolved the issues that it found.